source/GlassView.Export/Serialization/BenchmarkSerializer.cs

```diff
@@ -1,22 +1,21 @@
 using System.Text.Json;
 using GlassView.Core.Models;
-using GlassView.Export.Models;
 
 namespace GlassView.Export.Serialization;
 
-internal sealed class BenchmarkSerializer : System.Text.Json.Serialization.JsonConverter<Benchmark>
+internal sealed class BenchmarkSerializer : System.Text.Json.Serialization.JsonConverter<BenchmarkSummary>
 {
-    public override void Write(Utf8JsonWriter writer, Benchmark value, JsonSerializerOptions options)
+    public override void Write(Utf8JsonWriter writer, BenchmarkSummary value, JsonSerializerOptions options)
     {
         writer.WriteStartObject();
         WriteBenchmark(new JsonWriter(writer), value);
         writer.WritePropertyName(nameof(value.Environment));
         JsonSerializer.Serialize(writer, value.Environment, options);
         writer.WritePropertyName("BenchmarkCases");
-        JsonSerializer.Serialize(writer, (IEnumerable<IBenchmarkCase>)value, options);
+        JsonSerializer.Serialize(writer, (IEnumerable<BenchmarkCase>)value, options);
         writer.WriteEndObject();
 
-        static void WriteBenchmark(JsonWriter writer, IBenchmark value)
+        static void WriteBenchmark(JsonWriter writer, BenchmarkSummary value)
         {
             writer.Write(nameof(value.Name), value.Name);
```

Comment:
This seems very low level. Could you explain why is this required?

Reply:
See above :-)

Reply:
In that case, let's keep it as is 😄